### PR TITLE
Fix size auditor

### PR DESCRIPTION
**Pull request by ecraig12345**
Delete the accidental extra copy of ComponentConformance.test.tsx that was causing size auditor to fail after #15271.

And somewhat related changes:
- Update release notes to reflect changes from #15271 
- Clean up test-bundles package, including deleting stuff which appeared to only be used in a previous implementation of bundle size measurement. None of these things turned out to be the problem, but I made these changes while investigating.

### CODE REVIEW

**Comment by ecraig12345** [comment]
@msft-github-bot merge this in 1 minute

File: packages/react-next/RELEASE_NOTES.md [hunk 1]
```diff
@@ -117,6 +125,10 @@ Please see the [`@fluentui/react-theme-provider` package README](https://github.
 - `Button` and `Card` are new components that break from their previous implementation.
 - `WindowProvider` is required for child windows/embeds.
 - `FluentStyles` is removed from `experiments` package.
+- Removed various files which were originally in `office-ui-fabric-react` and not intended to be part of the public API:
+  - `office-ui-fabric-react/src/components/Theme/defaultTheme.ts`
+  - `office-ui-fabric-react/src/customizations/TeamsTheme.ts` (use `@fluentui/theme`)
```
Comment (xugao) [inline packages/react-next/RELEASE_NOTES.md:130]:
```suggestion
  - `office-ui-fabric-react/src/components/Theme/defaultTheme.ts` (use `@fluentui/theme`)
  - `office-ui-fabric-react/src/customizations/TeamsTheme.ts` (use `@fluentui/theme-samples`)
```

File: package.json [hunk 2]
```diff
@@ -71,7 +71,7 @@
     "danger": "^6.0.5",
     "gulp": "^4.0.2",
     "lerna": "^3.15.0",
-    "lage": "^0.19.0",
+    "lage": "^0.19.8",
```
Comment (xugao) [inline package.json:74]: just curious, why are you bumping this?

File: apps/test-bundles/webpackUtils.js [hunk 3]
```diff
@@ -85,7 +86,7 @@ function buildEntries(packageName, entries = {}, includeStats = true) {
 
   fs.readdirSync(packagePath).forEach(itemName => {
     const isJavascriptFile = itemName.match(/.js$/);
-    const isAllowedFile = TopLevelEntryFileExclusions.indexOf(itemName) === -1;
+    const isAllowedFile = TopLevelEntryFileExclusions.includes(itemName);
```
Comment (xugao) [inline apps/test-bundles/webpackUtils.js:89]:
```suggestion
    const isAllowedFile = !TopLevelEntryFileExclusions.includes(itemName);
```

File: apps/test-bundles/package.json [hunk 4]
```diff
@@ -1,45 +1,25 @@
...
-  "module": "lib/index.js",
-  "sideEffects": [
-    "lib/version.js"
-  ],
-  "typings": "lib/index.d.ts",
+  "description": "Testing Fluent UI React bundle sizes.",
   "private": true,
   "license": "MIT",
   "scripts": {
-    "build": "just-scripts build",
```
Comment (xugao) [inline apps/test-bundles/package.json:14]: we should keep `build`?

Comment (ecraig12345) [reply apps/test-bundles/package.json:14]: I'm pretty sure the only things to build in this package were the TS files for the old bundle size thing, which I didn't think were being used at all. Though now I see size auditor is consistently failing with weird lightrail errors, so maybe I was wrong about that?

File: package.json (hunk 2 above)
Comment (ecraig12345) [reply package.json:74]: I've been trying out `lage` locally and I wanted support for the `--to` flag (Ken said it was added more recently). Probably going to switch our local build commands to use it soon too.

File: apps/test-bundles/webpackUtils.js (hunk 3 above)
Comment (ecraig12345) [reply apps/test-bundles/webpackUtils.js]: ...oops 🤦‍♀️

File: apps/test-bundles/package.json (hunk 4 above)
Comment (ecraig12345) [reply apps/test-bundles/package.json:14]: Hmm the lightrail job is failing with similar errors on one of your PRs, so maybe it's unrelated https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=135115&view=results

**Review by xugao — APPROVED** [review]

**Pull request merged**